### DISCUSSION TITLE
404 File not found  The site configured at this address does not contain the requested file.

### DISCUSSION THREAD

**Discussion by melinalazaro** [community/community#132403] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello community, I have a problem. I have done the deploy of my site and everything is going well. But when I update some route I get the error: 
404
File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL as well as any file permissions.
For root URLs (like http://example.com/) you must provide an index.html file.

[Read the full documentation](https://help.github.com/pages/) for more information about using GitHub Pages.
Can you help me? 
my GitHub page is: https://github.com/melinalazaro/echuncano.git

tks

**Comment by LiteBrite82** [comment]
Hi @melinalazaro 

I was unable to replicate the issue. Looks like you got it working! 🎆 
https://melinalazaro.github.io/echuncano/

**Reply by melinalazaro** [reply]
the problem occurs when I am in a section and I want to refresh the page.  :(

**Reply by LiteBrite82** [reply, marked as answer]
Got it! Thats a router issue with react router. You need to use hashRouter in place of browserRouter. I found some helpful youtube tutorials that explain how and why to use hashRouter:

Why hashRouter instead of browserRouter
- [HashRouter vs. BrowserRouter | Significance of the HashRouter | Practical React by: GENIOBITS](https://www.youtube.com/watch?v=sRb-LJgtZ-k)

How to implement hashRouter
- [React Router Dom | Tutorial | HASHROUTER from {Bindev}](https://www.youtube.com/watch?v=2xQNhM2LZEk)

**Reply by kokaine11** [reply]
I have a similar challenge

https://kokaine11.github.io/Remote_E-Voting_System/

**Reply by melinalazaro** [reply]
super! thank you very much for your help
